src/DGtal/io/readers/GenericReader.ih

```diff
@@ -136,6 +136,26 @@ import( const std::string & filename,
 
 
 
+template <typename TContainer>
+inline
+TContainer
+DGtal::GenericReader<TContainer, 3, DGtal::uint64_t>::
+import( const std::string & filename) throw( DGtal::IOException )
+{
+  DGtal::IOException dgtalio;
+  const std::string extension = filename.substr( filename.find_last_of(".") + 1 );
+
+  if ( extension == "longvol" )
```

Comment:
I use sometime "lvol" for longvol extension.. I don't know if it is "standard"..